WooCommerce/WooCommerceTests/Model/CustomHelpCenterContentTests.swift

```diff
@@ -129,4 +129,24 @@ final class CustomHelpCenterContentTests: XCTestCase {
         XCTAssertEqual(sut.trackingProperties[CustomHelpCenterContent.Key.flow.rawValue], flow.rawValue)
         XCTAssertEqual(sut.trackingProperties[CustomHelpCenterContent.Key.url.rawValue], helpContentURL.absoluteString)
     }
+
+    // MARK: Store picker screen - `StorePickerViewController`
+    //
+    func test_init_screen_returns_valid_instance_for_store_picker_screen() throws {
+        // Given
+        let step = "site_list"
+        let flow: AuthenticatorAnalyticsTracker.Flow = .loginWithPasswordWithMagicLinkEmphasis
+        let helpContentURL = WooConstants.URLs.helpCenterForStorePicker.asURL()
```

Comment:
super nit: these feel more like in the "Then" section, since these aren't used for the setup but for expectations

Reply:
Makes sense. 👍 
Done in https://github.com/woocommerce/woocommerce-ios/pull/7641/commits/a91fe1a33dd07f5ed16e7d727c1c2d371c6ada0f